srcpkgs/lazarus/template

```diff
@@ -0,0 +1,35 @@
+# Template file for 'lazarus'
+pkgname=lazarus
+version=1.8.4
+revision=1
+wrksrc=lazarus
+#Dependencies only exist on x86_64, x86_64-musl, i686 and it does not build on x86_64-musl.
+only_for_archs="x86_64 i686"
+build_style=gnu-makefile
+makedepends="fpc qt5pas qt5-x11extras-devel"
+depends="fpc gdb fpc-source qt5pas"
+short_desc="Cross-platform IDE for FreePascal"
+maintainer="mobinmob <mobinmob@disroot.org>"
+license="GPL-2.0-or-later, LGPL-2.0-or-later, modified LGPL-2.0-or-later"
+homepage="https://www.lazarus-ide.org/"
+distfiles="https://sourceforge.net/projects/${pkgname}/files/Lazarus%20Zip%20_%20GZip/Lazarus%20${version}/${pkgname}-${version}.tar.gz"
+checksum=5a9801ac341a322ad1fa104d373f154d8ab72ba0add1693d0dcc8d936e6c10e9
+nocross=yes
+nopie=yes
```

Comment:
please comment on why these

Reply:
nocross: There are no cross-compilers for freepascal in the repo and no fpc outside i686 and x86_64, glibc/musl. 
nopie: Same as in lazarus-lcl-gtk2.

Reply:
Without nopie=yes, the build stops with:
`=> lazarus-1.8.4_1: running post-install hook: 06-strip-and-debug-pkgs ...
   Stripped executable: /usr/share/lazarus/components/chmhelp/lhelp/lhelp
=> ERROR: lazarus-1.8.4_1: non-PIE executable found in PIE build: /usr/share/lazarus/components/chmhelp/lhelp/lhelp                               
=> ERROR: lazarus-1.8.4_1: post-install_06-strip-and-debug-pkgs: 'find ${PKGDESTDIR} -type f' exited with 1                                       
=> ERROR:   in hook() at common/hooks/post-install/06-strip-and-debug-pkgs.sh:59                                                                  
=> ERROR:   in run_func() at common/xbps-src/shutils/common.sh:21        
=> ERROR:   in run_pkg_hooks() at common/xbps-src/shutils/common.sh:251  
=> ERROR:   in main() at common/xbps-src/libexec/xbps-src-doinstall.sh:67`

Ι can use nostrip=yes as a workaround instead of the (ugly) nopie. Is there a better solution?